frontend/packages/dev-console/src/components/formik-fields/_dropdown-field.scss

```diff
@@ -0,0 +1,6 @@
+.odc-dropdown-field {
+  &__inline-error {
+    color: --pf-global--danger-color--100;
+    font-style: 'italic';
+  }
+}
```

Comment:
Not needed because error message are handled already in the form group by patternfly.